artifacts/fedora/fedora.go

```diff
@@ -133,7 +134,9 @@ func (f *fedora) releaseMatches(release *Release) bool {
 }
 
 func (f *fedoraGatherer) releaseMatches(release *Release) bool {
-	return release.Arch == f.Arch &&
+	version, err := strconv.Atoi(release.Version)
+	return err == nil && version >= 35 &&
```

Comment:
Can you please introduce a [self-documenting](https://en.wikipedia.org/wiki/Self-documenting_code) constant here, e.g. `minimalFedoraVersion` ?